src/quo2/components/record_audio/record_audio/view.cljs

```diff
@@ -546,22 +599,43 @@
                                                                                     :linear))
            fade-out-animation        (fn []
                                        (reanimated/animate-shared-value-with-timing translate-x
-                                                                                    20
+                                                                                    (if @reviewing-audio?
+                                                                                      35
+                                                                                      20)
                                                                                     200
                                                                                     :linear)
+                                       (if @reviewing-audio?
+                                         (reanimated/animate-shared-value-with-timing scale
```

Comment:
you could have a local help functions with shorters names, just to simplify the code like `set-value`

Reply:
Done, added an internal helper namespace